pkg/vfs/vfsswift/impl_v1.go

```diff
@@ -102,20 +105,38 @@ func (sfs *swiftVFS) InitFs() error {
 }
 
 func (sfs *swiftVFS) Delete() error {
-	err := sfs.deleteContainer(sfs.version)
-	if err != nil {
-		sfs.log.Errorf("Could not delete version container %s: %s",
-			sfs.version, err.Error())
-		return err
+	containerMeta := swift.Metadata{"to-be-deleted": "1"}.ContainerHeaders()
+	sfs.log.Infof("Marking containers %q, %q and %q as to-be-deleted",
+		sfs.container, sfs.version, sfs.dataContainer)
+	err1 := sfs.c.ContainerUpdate(sfs.container, containerMeta)
+	err2 := sfs.c.ContainerUpdate(sfs.dataContainer, containerMeta)
+	err3 := sfs.c.ContainerUpdate(sfs.version, containerMeta)
+	if err1 != nil {
+		sfs.log.Errorf("Could not mark container %q as to-be-deleted: %s",
+			sfs.container, err1)
+		return err1
```

Comment:
Is there a reason why you removed the exact same `return` in impl_v2 and keep it here ?

Reply:
Mistake. Thanks for spotting it.